Source/Applications/SystemCenter/wwwroot/Scripts/TSX/SystemCenter/Settings/Setting.tsx

```diff
@@ -98,7 +98,7 @@ function Setting(props: IProps) {
         <>
             <LoadingScreen Show={status === 'loading'} />
             <div style={{ width: '100%', height: '100%' }}>
-                <SearchBar<SystemCenter.Types.Setting> CollumnList={searchFields} SetFilter={(flds) => dispatch(props.SettingsSlice.DBSearch({ filter: flds, sortField, ascending }))}
+                <SearchBar<SystemCenter.Types.Setting> StorageID={"SystemCenter.Setting.Search"} CollumnList={searchFields} SetFilter={(flds) => dispatch(props.SettingsSlice.DBSearch({ filter: flds, sortField, ascending }))}
```

Comment:
I believe that Component is used in OpenXDA Settings and SystemCenter Settings so they now share a key. In this case the Key needs to come in as `prop` to avoid that.